src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+  const message = document.createElement('div');
+
+  message.className = 'message';
+
+  const clickHandle = () => {
+    const text = 'Promise was resolved!';
+
+    message.append(text);
+
+    resolve(message);
+
+    logo.removeEventListener('click', clickHandle);
+  };
+
+  logo.addEventListener('click', clickHandle);
```

Comment:
If `logo` is `null`, this line will throw an error. Consider adding a check to ensure `logo` is not `null` before adding the event listener.